examples/analog_hamiltonian_simulation/README.md

```diff
@@ -2,4 +2,6 @@ Notebooks 00 and 05 introduce analog Hamiltonian simulation (AHS) and the AHS lo
 
 Notebook 01, 03 introduce QuEra's Aquila device and how to parallelize tasks on Aquila
 
-Notebook 02, 04, 06, 07 introduce several use cases for the Rydberg atom device
\ No newline at end of file
+Notebook 02, 04, 06, 07, 08 introduce several use cases for the Rydberg atom device
```

Comment:
Should this file have some formatting? At least should be "Notebooks... introduce... "